utilities/dotcms/locale.js

```diff
@@ -11,10 +12,34 @@ const locale = {
     }
 };
 
+export const setLocaleHref = ({ as, url = '', defaultLang }) => {
+    // If we have a selected language and the language is not the default lang
+    const currentLang = locale.getCurrentLanguage();
 
+    if (currentLang && currentLang !== defaultLang) {
+        /**
+         * if `url` is passed then return { as, url } to build the tags route
+         *
+         * @param {string} as e.g. '/store/category/[slug]'
+         * @param {string} url The path that we need to pass `/products/hello-world`
+         * @param {string} selectedLang The current selected language
+         */
```

Comment:
This is the doc of the function so it should be on top of the definition of the function.